lisa/tools/ethtool.py

```diff
@@ -0,0 +1,181 @@
+import re
+from pathlib import Path
+from typing import Any, List, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+from lisa.util import LisaException
+
+
+class DeviceChannel:
+    # ethtool device channel info is in format -
+    # ~$ ethtool -l eth0
+    #   Channel parameters for eth0:
+    #   Pre-set maximums:
+    #   RX:             0
+    #   TX:             0
+    #   Other:          0
+    #   Combined:       1
+    #   Current hardware settings:
+    #   RX:             0
+    #   TX:             0
+    #   Other:          0
+    #   Combined:       1
+    __current_settings_pattern = re.compile(
+        r"Current hardware settings:(\s+)(?P<settings>.*?)$", re.DOTALL
+    )
+
```

Comment:
Remove the blank line above, because the two patterns share the same example. And put max one above current one as their actual order.